torch/nn/modules/distance.py

```diff
@@ -31,7 +31,38 @@ def __init__(self, p=2, eps=1e-6):
     def forward(self, x1, x2):
         return F.pairwise_distance(x1, x2, self.norm, self.eps)
 
-# TODO: Cosine
```

Comment:
The Cosine is a different function, you should instead remove the `CosineDistance`